src/components/UpgradeTableOfContents.tsx

```diff
@@ -145,11 +115,28 @@ const UpgradeTableOfContents: React.FC<IProps> = ({
   }
 
   return (
-    <Aside className={className}>
-      <OuterList>
+    <Box
+      as="aside"
+      className={className}
+      p={0}
+      mb={8}
+      textAlign="end"
+      overflowY="auto"
+      display={{ base: "none", l: "block" }}
+    >
+      <UnorderedList
+        m={0}
+        py={0}
+        ps={4}
+        pe={1}
+        fontSize="1.25rem"
+        fontWeight="normal"
+        lineHeight="1.6"
```

Comment:
No variables for this, Chakra defaults don't cover 1.6.

Reply:
In this case, we should just use the raw value: `1.6rem`.

Reply:
Another thought I had, could you try to remove it and test if it works? because we are defining in [the global styles that we are using `1.6rem` as the default](https://github.com/ethereum/ethereum-org-website/blob/dev/src/%40chakra-ui/gatsby-plugin/styles.ts#L20).